content/en/blog/_posts/2021-06-21-update-with-ingress-nginx.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: blog
+title: 'Updating NGINX-Ingress to use the stable Ingress API'
+date: 2021-06-21
+slug: update-with-ingress-nginx
+---
```

Comment:
How about we get this merged _as a draft_?
Then, when we want to publish it, it'll be a small diff. We can also accept PRs to keep the draft current during the period before we can get that published.

```suggestion
---
draft: true
layout: blog
title: 'Updating NGINX-Ingress to use the stable Ingress API'
date: 2021-06-21
slug: update-with-ingress-nginx
---
```

Reply:
updated.